libs/ledger-live-common/src/families/hedera/utils.ts

```diff
@@ -2,13 +2,24 @@ import estimateMaxSpendable from "./js-estimateMaxSpendable";
 import BigNumber from "bignumber.js";
 import type { Account } from "@ledgerhq/types-live";
 import type { Transaction } from "./types";
+import axios from "axios";
```

Comment:
Not mandatory, but you should use our `network` util if possible, the main benefit is that it uses our logs()